core/src/main/scala/kafka/server/ConfigAdminManager.scala

```diff
@@ -369,15 +365,13 @@ object ConfigAdminManager {
     persistentResponses: AlterConfigsResponseData
   ): AlterConfigsResponseData = {
     val response = new AlterConfigsResponseData()
-    val responsesByResource = persistentResponses.responses().iterator().asScala.map {
-      case r => (r.resourceName(), r.resourceType()) -> new ApiError(r.errorCode(), r.errorMessage())
```

Comment:
Previous code was the standard in really old version of Scala, since then anonymous function style is preferred.